taskchampion/src/task/task.rs

```diff
@@ -151,6 +165,16 @@ impl Task {
         self.taskmap.contains_key(Prop::Start.as_ref())
     }
 
+    /// Determine whether this task is blocked -- that is, has at least one unresolved dependency.
+    pub fn is_blocked(&self) -> bool {
+        self.depmap.dependencies(self.uuid).next().is_some()
```

Comment:
Huh, kind of surprising the iterator doesn't have an equivelant of an `is_empty()` method (`.count() == 0` is the next closest thing, but `.next().is_some()` is probably more efficient)

Reply:
Yeah, I went looking for such a thing.  The docs mention "empty iterator" lots of times, but no "is_empty".  I suspect that's because it's not possible to do non-destructively?